pages/cathocomponents.js

```diff
@@ -29,116 +29,132 @@ export default function CathoComponentsPage() {
       <Head>
         <title>Catho Components - SSR-Simulator</title>
       </Head>
-      <Container className=" ColoredBackground Margins">
-        <h4 className="TextFormattingH4">Headers</h4>
-        <h6>
-          These are responsive Headers for all Catho applications. They follow
-          Catho&apos;s style guide and its needs.
-        </h6>
-        <h5 className="TextFormattingH5">Header Responsive</h5>
-        <HeaderResponsive
-          type="B2C"
-          isSubMenuOpen={false}
-          gtmPrefix="next-header-b2c-nosearch"
-          hasSearch
-        />
-        <h5 className="TextFormattingH5">Logged Header Responsive</h5>
-        <HeaderLoggedResponsive userProfile="STATUS_I_PROFISSIONAL" hasSearch />
-        <h5 className="TextFormattingH5">Simple Header</h5>
-        <HeaderSimple gtmPrefix="next" />
-      </Container>
-      <Container className=" FlexContainer ColoredBackground Margins">
-        <Row>
-          <Col
-            large={6}
-            medium={4}
-            small={4}
-            xmall={2}
-            className=" RoundedEdges WhiteBackground  PaddingLeftRight"
-          >
-            <h4 className="TextFormattingH4">Password Field</h4>
+      <main>
+        <section>
```

Comment:
Here and on the quantum page there are several sections. I see the quantum and catho-components page with an "examples section" instead of "example sections". Therefore, I believe that using a single section as the parent element of all examples is more interesting, what do you think?